x-pack/legacy/plugins/maps/server/maps_telemetry/collectors/register.ts

```diff
@@ -4,18 +4,25 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { getMapsTelemetry } from './maps_telemetry';
-import { TELEMETRY_TYPE } from '../../common/constants';
+import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
+// @ts-ignore
+import { getMapsTelemetry } from '../maps_telemetry';
+// @ts-ignore
+import { TELEMETRY_TYPE } from '../../../common/constants';
 
-export function initTelemetryCollection(usageCollection, server) {
+export function registerMapsUsageCollector(
+  usageCollection: UsageCollectionSetup,
+  savedObjectsClient: any,
```

Comment:
Don't we know the exact type here? https://github.com/elastic/kibana/blob/ec2134d221aadc732c30a7bc88953cebcdf23c6f/src/core/public/saved_objects/index.ts#L26

We should import it from `kibana/public` or `src/core/public`. 

e.g. https://github.com/elastic/kibana/blob/d6a500cdb3599e74ea0f06cb935329a41ef59378/x-pack/legacy/plugins/lens/public/app_plugin/plugin.tsx#L20


In general, we should start avoiding the use of `any`.

If we really really have to, we should type with `unknown` and doing a type-check before using the variable.